views/review/orders-recent-modify.html.twig

```diff
@@ -0,0 +1,78 @@
+{% extends 'base.html.twig' %}
+{% block title %}Orders Recent Modify - Participant Review - {% endblock %}
+{% block bodycontainer %}container-fluid{% endblock %}
+
+{% macro displayDate(order, dateType) %}
+    {% if order[dateType] %}
+        {{ order[dateType]|date('n/j/Y g:ia', app.userTimezone) }}
+    {% else %}
+        <i class="fa fa-times text-danger" aria-hidden="true"></i>
+    {% endif %}
+{% endmacro %}
+
+{% import _self as today %}
+
+{% block body %}
+    <div class="page-header">
+        <h2><i class="fa fa-list" aria-hidden="true"></i> Participant Review</h2>
+    </div>
+    <ul class="nav nav-tabs">
+        <li role="presentation"><a href="{{ path('review_today') }}">Today's Participants</a></li>
+        <li role="presentation"><a href="{{ path('review_orders') }}">Unfinalized Biobank Orders</a></li>
+        <li role="presentation"><a href="{{ path('review_measurements') }}">Unfinalized Physical Measurements</a></li>
+        <li role="presentation" class="active"><a href="#">Recently Modified Biobank Orders</a></li>
+    </ul>
+    <br/>
+    <div class="container">
+        <table class="table table-striped table-bordered">
+            <thead>
+            <tr>
+                <th>Order ID</th>
+                <th>Created</th>
+                <th>Collected</th>
+                <th>Processed</th>
+                <th>Finalized</th>
+                <th>Modified</th>
+                <th>Status</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for order in orders %}
+                <tr>
+                    <td>
+                        <a href="{{ path('order', { participantId: order.participant_id, orderId: order.id }) }}">{{ order.order_id }}</a>
+                    </td>
+                    <td>
+                        {{ today.displayDate(order, 'created_ts') }}
+                    </td>
+                    <td>
+                        {{ today.displayDate(order, 'collected_ts') }}
+                    </td>
+                    <td>
+                        {{ today.displayDate(order, 'processed_ts') }}
+                    </td>
+                    <td>
+                        {% if order.oh_type != 'unlock' %}
+                            {{ today.displayDate(order, 'finalized_ts') }}
+                        {% else %}
+                            <i class="fa fa-times text-danger" aria-hidden="true"></i>
+                        {% endif %}
+                    </td>
+                    <td>
+                        {{ today.displayDate(order, 'oh_created_ts') }}
+                    </td>
+                    <td align="center">
+                        {% if order.oh_type == 'edit' %}
+                            <span class="label label-success">Edited & Finalized</span>
```

Comment:
[Trivial] GitHub's syntax highlighter complains that this isn't `&amp;` instead of `&`.